source/wp-content/themes/wporg-developer-2023/templates/single.html

```diff
@@ -1,3 +1,5 @@
 <!-- wp:template-part {"slug":"header","className":"has-display-contents"} /-->
-single
+
+<!-- wp:pattern {"slug":"wporg-developer-2023/single-content"} /-->
```

Comment:
This template is also used for all other "single" pages - ex, handbook pages, class reference, etc. For now this works, but we'll probably want to use a specific template for the code reference pages, and let this be the fallback for handbook-style pages (and that will probably look more like the [documentation single.html](https://github.com/WordPress/wporg-documentation-2022/blob/trunk/source/wp-content/themes/wporg-documentation-2022/templates/single.html).

Reply:
I guess we should also make sure that `single-{post-type}.html` works in block themes.

Reply:
I played with this a bit for handbooks but with moderate success, I can get the template to load for the specific handbooks but for some, the content is not loaded.